tests/test_users.py

```
@@ -52,9 +61,12 @@ def test_user_list():
 
     # get list
     l = UserList.get(data['name'], sean.username)
```

Ruff (0.15.13)

[error] Ambiguous variable name: `l`

(E741)